config/ompi_check_openfabrics.m4

```diff
@@ -168,15 +168,19 @@ AC_DEFUN([OMPI_CHECK_OPENFABRICS],[
            # ibv_cmd_open_xrcd (aka XRC Domains) was added in  OFED 3.12
            if test "$enable_connectx_xrc" = "yes"; then
                $1_have_xrc=1
-               AC_CHECK_FUNCS([ibv_create_xrc_rcv_qp ibv_cmd_open_xrcd],
+               AC_CHECK_FUNCS([ibv_create_xrc_rcv_qp],
                               [], [$1_have_xrc=0])
-               AC_CHECK_DECLS([IBV_SRQT_XRC],
```

Comment:
IBV_SRQT_XRC is specific to XRC domains, that came with OFED 3.12, so the this test disables XRC on OFED < 3.12 :-(
Instead, we can test IBV_DEVICE_XRC that is common to all OFEDs, but is not part of the RHEL6 broken headers